build.gradle

```diff
@@ -24,8 +24,8 @@ java {
     }
 
     compileJava {
-        // Set binary compatibility for Java compiler to 11, as it's used as a convention across all projects
-        options.release.set(11)
+        // Set binary compatibility for Java compiler to 17
+        options.release.set(17)
```

Comment:
Have you tried running this with the platform? You should bump the version on migtool to 1.5.1 and build It with `./gradlew publishToMavenLocal` and then bing local project in the platform. If you haven't done It yet we can try It together.

Reply:
Thanks @weronikasosnowskaseqera. 

No, I haven't tried that yet. Basically only run the tests, so this needs more testing still.
Just tell me when you have time so we can do it together then!